### PR TITLE
Increase ruby support to 2.6

### PR DESCRIPTION
**Pull request by gleopoldo**
Looking at the travis CI, I noticed that we don't test the gem's test suite against newer ruby versions.

I'm adding them to the CI now, so that we will have more confidence that we're not introducing unsupported/legacy code.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,3 +22,5 @@ matrix:
     - rvm: "2.3.0"
     - rvm: "2.3.3"
     - rvm: "2.4.1"
+    - rvm: "2.5.7"
+    - rvm: "2.6.5"
```
Comment (fabioperrella) [inline .travis.yml:26]: humm, that's a great tip to open some PRs for hacktoberfest!

Comment (fabioperrella) [inline .travis.yml:26]: What about changing the gemspec file and set the minimum supported ruby version?

**Review by fabioperrella — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (gleopoldo) [reply .travis.yml:26]: I was thinking about this too... Do you think that we can already remove support for versions that reached their EOL?

Comment (fabioperrella) [reply .travis.yml:26]: Humm, I think it is not a good idea since I know some apps that are using these old ruby versions

**Pull request merged**